_includes/nav_v1.0.html

```diff
@@ -5,8 +5,8 @@
 	<li class="navContainer">
 		<a href="{{ site.url }}/v1.0/docs/whatisk8s.html" class="nav-link">What is Kubernetes?</a>
 	</li><!-- 
-					GETTING STARTED
-					 -->
+GETTING STARTED
```

Comment:
It's a minor point but both the before and after indenting is weird on these.  Is there a reason these one-word comments are split across three lines?